src/main/java/seedu/address/model/candidate/InterviewStatus.java

```diff
@@ -8,7 +8,7 @@
     public static final String INTERVIEWING_STATUS = "Interviewing";
     public static final String NOT_INTERVIEWED_STATUS = "Not Interviewed";
 
-    private String interviewStatus;
+    public final String interviewStatus;
```

Comment:
Perhaps this one should remain as private - just to have a better level of abstraction. But smol change, I can do it in the next PR ! Thanks dom!

Reply:
yes please!